### ISSUE TITLE
[feature] - include a closing "ta-da" summary

### ISSUE THREAD

**Issue by alexgroom** [redhat-scholars/backstage-workshop#15]
When you get to the end of the workshop guide there is no ta-da, or wow moment, it just stops on a curl command.

There should be at least a conclusion, a summary of what has been achieved, a Why? reminding how good the user should be feeling about completing this workshop. etc

**Comment by joshix** [comment]
Agree. We will add context both within the exercises (what backstage is doing, so you have a sense of work saved) and in a conclusion/summary and CTA type stuff at the end.

**Comment by ryanj** [comment]
I added a brief closer targeting our reviewers - this section will definitely need to be updated again before Summit

**Comment by joshix** [comment]
related to #33
